src/test/java/org/jenkinsci/plugins/androidsigning/SignApksStepTest.java

```diff
@@ -54,8 +55,13 @@ public void setupEnvironment() throws Exception {
         slave.setLabelString(slave.getLabelString() + " " + getClass().getSimpleName());
     }
 
+    @AfterEach
+    void afterEach() {
+        testKeyStore.removeCredentials();
+    }
```

Comment:
While we could make `TestKeyStore` an `Extension` (the JUnit Jupiter equivalent of a `Rule`) there is little reason of doing so (see comments in TestKeyStore.java). Instead we call `#addCredentials()` in `beforeEach()` and `#removeCredentials()` in `afterEach()`. Is makes it also easier to initialize the object with a `JenkinsRule` which becomes tricky here due to the design of the rule.